### PR TITLE
Doc release 2.4.0

### PR DESCRIPTION
**Pull request by codelipenghui**
### Motivation

Release 2.4.0 doc

### CODE REVIEW

**Comment by codelipenghui** [comment]
run java8 tests

File: site2/website/versioned_docs/version-2.4.0/io-cdc-canal.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+```
+docker pull canal/canal-server:v1.1.2
+docker run -d -it --link pulsar-mysql -e canal.auto.scan=false -e canal.destinations=test -e canal.instance.master.address=pulsar-mysql:3306 -e canal.instance.dbUsername=root -e canal.instance.dbPassword=canal -e canal.instance.connectionCharset=UTF-8 -e canal.instance.tsdb.enable=true -e canal.instance.gtidon=false --name=pulsar-canal-server -p 8000:8000 -p 2222:2222 -p 11111:11111 -p 11112:11112 -m 4096m canal/canal-server:v1.1.2
+```
+
+- Start pulsar standalone
+
+```$bash
+docker pull apachepulsar/pulsar:2.3.0
```
Comment (jiazhai) [inline site2/website/versioned_docs/version-2.4.0/io-cdc-canal.md]: Seems here 2.3.0 should be replaced?

File: site2/website/versioned_docs/version-2.4.0/io-cdc-canal.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+```$bash
+docker cp canal-mysql-source-config.yaml pulsar-standalone:/pulsar/conf/
+docker cp pulsar-client.py pulsar-standalone:/pulsar/
+```
+
+- Download canal connector and start canal connector
+```$bash
+docker exec -it pulsar-standalone /bin/bash
+wget http://apache.01link.hk/pulsar/pulsar-2.3.0/connectors/pulsar-io-canal-2.3.0.nar -P connectors
```
Comment (jiazhai) [inline site2/website/versioned_docs/version-2.4.0/io-cdc-canal.md]: also here for 2.3.0. and the apache download link(http://apache.01link.hk) would be better replaced to a dynamic one?

**Comment by codelipenghui** [comment]
run java8 tests

**Comment by codelipenghui** [comment]
@jiazhai I have addressed your comments, PTAL.

**Comment by codelipenghui** [comment]
run java8 tests

**Comment by codelipenghui** [comment]
run java8 tests

**Comment by jiazhai** [comment]
👍

**Pull request merged**